okhttp/src/main/kotlin/okhttp3/ConnectionSpec.kt

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (C) 2014 Square, Inc.
+ * Copyright (C) 2024 Square, Inc.
```

Comment:
deliberate?

Reply:
No, that's weird. I set up automatic file headers in IJ, it must have decided to update some existing files sometimes... 🤔 

Anyway, will revert